### PR TITLE
Start selling Marionette on the homepage

### PR DESCRIPTION
**Pull request by jasonLaster**
I'm sick of us not talking about why Marionette is great. Well, I decided I wasn't going to take it any more and that it was going to be fixed.

@jdaudier I need your help w/ this copy. Actually all y'all help would be great @jmeas, @samccone, @thejameskyle

### CODE REVIEW

File: src/sections/_about.jade [hunk 1]
```diff
@@ -1,28 +1,58 @@
...
+      You can think of Marionette in one of two ways.
+      Marionette has the most powerful building blocks for building a Backbone application.
+      Marionette is the collective wisdom of the most inclusive vibrant Backbone community.
+
+  .features
+    .feature.col-1-3.left
+      header
+        h2 Layouts
+      p.description.
+        Organize your app in terms of small views.
```
Comment (samccone) [inline src/sections/_about.jade]: Organize your app in terms of ~~small~~ compossible declarative  views.

**Comment by samccone** [comment]
I would love to get some Icons for each of these

ping @mrossimojo  @lupinskij

File: src/sections/_about.jade [hunk 2]
```diff
@@ -1,28 +1,58 @@
...
+        h2 Behaviors
+      p.description.
+        Share complex ui interactions across views.
+        Behaviors are mixins, without all of the hassle of property collision.
+
+    .feature.col-1-3.left
+      header
+        h2 Radio and Channels
+      p.description.
+        Communicate between independent applications with a powerful evented system.
```
Comment (samccone) [inline src/sections/_about.jade]: system and architecture

File: src/sections/_about.jade [hunk 3]
```diff
@@ -1,28 +1,58 @@
...
+
+    .feature.col-1-3.left
+      header
+        h2 Objects
+      p.description.
+        Write your application classes with Objects. You'll get
```
Comment (samccone) [inline src/sections/_about.jade]: Embrace the Marionette evented object lifecycle by extending your code with our base classes.

File: src/sections/_about.jade [hunk 4]
```diff
@@ -1,28 +1,58 @@
...
+  .features
+    .feature.col-1-3.left
+      header
+        h2 Layouts
+      p.description.
+        Organize your app in terms of small views.
+
+    .feature.col-1-3.left
+      header
+        h2 Lists
```
Comment (samccone) [inline src/sections/_about.jade]: i would love to drop this or view helpers and replace it with "hackable"

Confused how something works? read the annotated source code. Marionette prides itself on having a readable source code approachable at all levels

File: src/sections/_about.jade [hunk 5]
```diff
@@ -1,28 +1,58 @@
...
+        h2 View Helpers
+      p.description.
+        Learn to love the little details.
+        We've added tons of features to our views. You'll never want to go back.
+
+    .feature.col-1-3.left
+      header
+        h2 Behaviors
+      p.description.
+        Share complex ui interactions across views.
```
Comment (samccone) [inline src/sections/_about.jade]: Abstract and reuse complex ui interactions across your views. 
Compose your application interaction through the powerful Marionette only concept.

File: src/stylesheets/_about.scss [hunk 6]
```diff
@@ -63,6 +69,8 @@
   }
 }
 
+
```
Comment (samccone) [inline src/stylesheets/_about.scss]: :x: newlines or whatever

**Comment by jasonLaster** [comment]
After working with sam for a bit, we put some additional content in:



I think this is enough to begin thinking about what is the sharpest message to leave people with.

**Comment by jasonLaster** [comment]
Third attempt... So the second attempt in my mind had more content, but was less cohesive so i broke the content into two sections: functionality and values:

**Comment by jdaudier** [comment]
I wonder if we need a heading for each section... like

### What Marionette Gives You

and

### Why Marionette?

**Comment by jasonLaster** [comment]
@jdaudier I think you're right.

**Comment by samccone** [comment]
@jasonLaster @jdaudier want to make a google doc and we can iterate in there? I think it will be better than githubs format for copy writing

**Comment by samccone** [comment]
i want to get this sucker in, it is so so much better than what we have and I think will do just what you want Jason... SELL marionette :moneybag:

**Comment by benmccormick** [comment]
@samccone asked me for my thoughts on the copy for this, so here goes:

- The values section in the last version is pretty much perfect.  Great summary of the things that attracted me to Marionette.  
- I would make the layouts' copy something like: "Organize your app in terms of small Views.  Marionette makes it easy to compose rich layouts out of small components".  You want to catch the layout abilities, not just the modularity.
- In the Radio section "communicate across independent applications" is confusing.  You presumably mean Mn "Applications", but people unused to Mn may assume you mean actual web applications and start thinking about cross domain requests and other craziness. How about "Decoupled communication between your application components with a powerful messaging system"
- I really like "write classes with the same API as your views" as an explanation for Mn.Object

There are a few grammar cop notes too, but I'll just inline those in the code :)

File: src/sections/_about.jade [hunk 7]
```diff
@@ -1,28 +1,90 @@
...
+
+        Have a massive collection? Want to add or remove an item? No worries!
+
+    .feature.col-1-3.left
+      header
+        h2 Utilities
+      p.description.
+        Learn to love the details.
+
+        We've added tons of features that will keep you from ever
```
Comment (benmccormick) [inline src/sections/_about.jade]: ever wanting ~~going back~~ to go back.

File: src/sections/_about.jade [hunk 8]
```diff
@@ -1,28 +1,90 @@
...
+        h2 Radio
+      p.description.
+        Communicate between independent applications with a
+        powerful evented system.
+
+    .feature.col-1-3.left
+      header
+        h2 Objects
+      p.description.
+        Write classes with the same api as your views.
```
Comment (benmccormick) [inline src/sections/_about.jade]: api should be API, its an abbreviation.

File: src/sections/_about.jade [hunk 9]
```diff
@@ -1,28 +1,90 @@
...
+      p.description.
+        Marionette community is home to the most
+        welcoming and vibrant discussions in the Backbone ecosystem.
+    .cl
+
+    .feature.col-1-2.left
+      header
+        h2 Iterative
+        p.description.
+          Have a large unruley code base that you can not simply rewrite?
```
Comment (benmccormick) [inline src/sections/_about.jade]: unruley => unruly

**Comment by samccone** [comment]
kk I am going to fix this up based on the feedback and :shipit: tonight 

unless any objections

**Comment by peterblazejewicz** [comment]
:+1:  LGTM

**Comment by jdaudier** [comment]
Just skimmed over the copy.  The only thing that was not very clear to me was the Utilities section. What did you mean by that @jasonLaster?

**Comment by benmccormick** [comment]
I think the Utilities section was clearer in the earlier revisions when it explicitly mentioned Views.
I'm assuming you meant stuff like the ui hash, model/collection events, triggers, templateHelpers, default renderer, etc right @jasonLaster ?

**Comment by samccone** [comment]
thanks for the feedback @benmccormick and @jdaudier 
I have adjusted both of these sections and am shipping it all now.

We can continue to iterate from here.

@peterblazejewicz with this new copy I think there are going to be some opportunities for some better schema tags and or declarative DOM nodes to make this content more digestible from a robots perspective.


thanks all for the amazing work :clap: :clapper: :baggage_claim:

**Comment by peterblazejewicz** [comment]
not robots - first people using readers, etc, then robots.
The page reads now:
https://gist.github.com/peterblazejewicz/dd6402f7254bee2e38a9

**Comment by samccone** [comment]
Seems like we might want to add some section headers on each of those blocks, what do you think @peterblazejewicz

**Comment by peterblazejewicz** [comment]
@samccone not necessarily - the main problem I could foresee with updated copy is that there is no prominent clear clear text paragraph now in top of the page:
```
The Backbone Framework
 
   Marionette simplifies your Backbone application code with robust views and architecture solutions.
```
Can be too short and Google and others can pick different text from page. But we have addes schemas as remedy for this - so we will see.
The plain text: https://gist.github.com/peterblazejewicz/dd6402f7254bee2e38a9
let us see what we are writing on our page. Current sections are excellent starting point to put links to other page(s) with details about given section topic (`oh, "Layouts" - how can I manage my layout - let's see` [click]).

**Comment by benmccormick** [comment]
Section headers might be good for mobile even if they're not needed for accessibility.

I love the new layout on desktop, but on mobile the hierarchy of the page is lost and you get a long list of things before any call to action or context 

marionette_js_ _the_backbone_framework_and_untitled

**Pull request closed without merging**